### DISCUSSION TITLE
Welcome to leetcode-editor Discussions!

### DISCUSSION THREAD

**Discussion by shuzijun** [shuzijun/leetcode-editor#378] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qi-yun** [comment]
在idea上配置leetcode-editor8.0版本只会出现每日一题，不会出现这些题型分类：
![1650518441(1)](https://user-images.githubusercontent.com/46617275/164378475-965a33eb-9be9-41de-a502-c0bbb1a3399c.png)
建议https://plugins.jetbrains.com/plugin/12132-leetcode-editor/versions上面下载leetcode-editor7.3版本，可出现分类题型

**Comment by tuoruipeng** [comment]
如何使用第三方账号登录？

**Reply by shuzijun** [reply]
> 如何使用第三方账号登录？

只配置用户名,可以弹出登录网页，在网页里使用三方登录即可

**Reply by tuoruipeng** [reply]
还是不行哎，显示登录失败，也没有弹出登录网页

**Reply by shuzijun** [reply]
> 还是不行哎，显示登录失败，也没有弹出登录网页

使用的版本是多少

**Reply by tuoruipeng** [reply]
v6.9

**Reply by shuzijun** [reply]
> v6.9

在配置里启用JCEF，需要IDE支持才可以

**Reply by tuoruipeng** [reply]
解决了，感谢！
